### DISCUSSION TITLE
Form组件如何按指定规则进行校验？

### DISCUSSION THREAD

**Discussion by vfan140** [ant-design/ant-design#52036] [General]
现在这边有个诉求，希望在某些场景下提交表单的时候，只要校验表单中某类规则，如我有一个自定义校验器{ type: 'abc', validator: …… }，希望在提交的时候表单控件只校验type='abc'的规则。

目前form.validateFields的第二个参数支持传递validateOptions，但是validateOptions的选项有限，并不能完成按指定规则校验，请问是否有别的方法完成这样的诉求。

**Comment by Wxh16144** [comment]
https://ant.design/components/form-cn#form-demo-dynamic-rule 这个可以满足么
